MovieLibrary/src/com/company/FileUtils.java

```diff
@@ -0,0 +1,88 @@
+package com.company;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileUtils {
+
+
+    private static final String COMMA_DELIMITER = ",";
+
+    public static <T> List<T> readFromFileStream(String fileName) {
+        List<T> result = new ArrayList<>();
+        try {
+            FileInputStream fileInputStream = new FileInputStream(fileName);
+            ObjectInputStream objectInputStream = new ObjectInputStream(fileInputStream);
+            while (fileInputStream.available() > 0) {
+                T readObject = (T) objectInputStream.readObject();
+                result.add(readObject);
+            }
+            fileInputStream.close();
+        } catch (IOException | ClassNotFoundException e) {
+            e.printStackTrace();
+            System.out.println(e.getMessage());
+        }
+        return result;
+    }
+    public static boolean fileExists(String fileName) {
+        File file = new File(fileName);
+        return file.exists();
+    }
+
+    public static boolean createFile(String fileName) {
+        File file = new File(fileName);
+        try {
+            return file.createNewFile();
+        } catch (IOException e) {
+            System.out.println("cannot create file: " + fileName);
+        }
+        return false;
+    }
+
+    public static void csvWriter(List<Movie> movies) throws IOException {
+        BufferedWriter out = new BufferedWriter(new FileWriter("csvMovieList"), 512);

```

Comment:
"csvMovieList" to tzw. Magic string.

Teksty powinny trafiać do zmiennych, lub (w tym przypadku) do prywatnych stałych w klasie

Reply:
Do zastosowania w całym projekcie